Documentation/usage/dlv_debug.md

```diff
@@ -13,7 +13,7 @@ package name and Delve will compile that package instead, and begin a new debug
 session.
 
 ```
-dlv debug [package]
+dlv debug [package] -- [package arguments]
```

Comment:
These documentation files are autogenerated by scripts/gen-cli-docs.go and shouldn't be changed directly. There's a test that's supposed to check that this doesn't happen but it was broken.

Reply:
Ahh okay. I’ll pull that commit today and leave the version update. Thanks!